server/asset/btc/rpcclient.go

```diff
@@ -22,6 +22,7 @@ const (
 	methodGetTxOut          = "gettxout"
 	methodGetRawTransaction = "getrawtransaction"
 	methodGetBlock          = "getblock"
+	methodGetIndexInfo      = "getindexinfo"
```

Comment:
Since this method was [introduced in Bitcoin Core v0.21](https://bitcoincore.org/en/releases/0.21.0/#new-rpcs), that officially breaks support for v0.20, which people still use.  It also is a problem for other forks of Bitcoin Core (e.g. Litecoin 0.18) that don't have this either.

This could be resolved in different ways.  Simplest is to just log a warning, which may or may not be seen by the operator.  We could detect a "method not found" error and ignore the error.  More complete would be to check the version like we do on client side.